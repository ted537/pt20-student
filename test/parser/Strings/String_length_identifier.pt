using input;
val str = "Hello World";
var length = #str;

