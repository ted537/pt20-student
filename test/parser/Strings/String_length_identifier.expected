 .sProgram
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
     .sConst
      .sIdentifier
      .sStringLiteral
    % .sNewLine
    .sVar
     .sIdentifier
          .sIdentifier
         .sLength
    % .sNewLine
    % .sNewLine
  .sEnd
